### PR TITLE
Missing documentation for `as:`

### PR DESCRIPTION
**Pull request by justin808**
See https://github.com/rails/jbuilder/issues/385

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +170,25 @@ json.partial! partial: 'posts/post', collection: @posts, as: :post
 json.comments @post.comments, partial: 'comments/comment', as: :comment
 ```
 
+The `as: :some_symbol` is used with partials. It will take care of mapping the passed in object to a variable for the partial. If the value is a collection (either implicitly or explicitly by using the `collection:` option, then each value of the collection is passed to the partial as the variable `some_symbol`. If the value is a singular object, then the object is passed to the partial as the variable `some_symbol`. 
+
+Be sure not to confuse the `as:` option to mean nesting of the partial. For example:
+
+```ruby
+ # Use the default `views/comments/_comment.json.jbuilder`, putting @comment as a the comment local variable
```
Comment (rafaelfranca) [inline README.md]: `as the`

Comment (rafaelfranca) [inline README.md]: Missing `.` in the end of the sentence

Comment (justin808) [inline README.md]: Fixed

Comment (justin808) [inline README.md]: Fixed

**Comment by justin808** [comment]
@rafaelfranca Can we get this merged? I fixed this 19 days ago.

**Comment by justin808** [comment]
@rafaelfranca Anything I can do to help you get this merged?

**Comment by justin808** [comment]
@rafaelfranca Shall we merge this?

**Comment by jpgeek** [comment]
Any reason this has not been merged?  Are pull requests no longer being accepted for some reason?

**Comment by olleolleolle** [comment]
@kaspth If I may bother you for a review of this **documentation addition**, which has been reviewed once - the feedback seems to have been met.

**Comment by kaspth** [comment]
Thanks @justin808, sorry for the long wait! Sometimes it's just one of those things 😊

**Pull request merged**